server/server.js

```diff
@@ -0,0 +1,41 @@
+/* eslint no-console:0 */
+var loopback = require('loopback');
+var boot = require('loopback-boot');
+var ServiceManager = require('./service-manager');
+
+function server(serviceManager) {
```

Comment:
This is a constructor, why isn't it capitalized? I understand `loopback()` and `express()` aren't capitalized (for some historical reason, I assume, they are clearly constuctors, not sure what's going on there). I noticed this in the export, where meshServer is exported, which gave me pause: was a server *instance* being exported, or a constructor for a server? The naming implied the former, but I expected the latter. So, I think the naming convention matters.